sway/tree/container.c

```diff
@@ -1404,15 +1404,15 @@ enum sway_container_layout container_current_parent_layout(
 }
 
 list_t *container_get_siblings(struct sway_container *container) {
-	if (container->pending.parent) {
-		return container->pending.parent->pending.children;
-	}
-	if (container_is_scratchpad_hidden(container)) {
+	if (container_is_scratchpad_hidden_or_child(container)) {
```

Comment:
Returning NULL for children of scratchpad hidden split containers may crash in certain operations. See comment.